doc/field_guide__lpv.md

```diff
@@ -0,0 +1,43 @@
+## Long Period Variables (lpv)
+Long Period Variables have periods over several tens of days.
+They incorporate various groups of cool luminous pulsators.
+The canonical example is Mira.
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+  - pulsator
+- Subtypes
+  - Miras
+  - Semiregulars
+  - L stars
+  - RV Tau
+- Occurrence rate: very common, about 10<sup>5</sup> expected in ZTF data
+
+
+### ZTF light curves
+![ZTF LPV](data/lpv.png)
+
+#### Description
+Long period variables have periods from several tens of days to hundreds of days and are
+easy to identify due to the slow varioations.
+
+#### Light curve characteristics
+- periodic variable
+- period range: 100-1000 days

```

Comment:
You are saying above that it is tens of days - ...

Reply:
I feel tens of days is consistent with hundred. May be several tens of days. Will tighten up the wording when I make more changes.